<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M_SetTrigSingal</class>
 <widget class="QDialog" name="M_SetTrigSingal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pulse</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>421</width>
     <height>305</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_all">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>405</width>
      <height>212</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>195</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QCheckBox" name="checkBox_path0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>14</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_path1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>53</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_path2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>93</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_path3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>132</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_path4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>173</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 4</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_path0">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>10</y>
       <width>311</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_0_width">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_0_delay">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox__path1">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>50</y>
       <width>311</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_1_width">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_1_delay">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_path2">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>90</y>
       <width>311</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2_width">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2_delay">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_path3">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>130</y>
       <width>311</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3_width">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3_delay">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_path4">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>170</y>
       <width>311</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4_width">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4_delay">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>405</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_period">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>13</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_pulse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>12</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pulse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>88</x>
       <y>14</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Period：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>265</y>
      <width>405</width>
      <height>33</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_ok">
     <property name="geometry">
      <rect>
       <x>303</x>
       <y>3</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_default">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>4</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
